README.md

```diff
@@ -183,7 +202,40 @@ go build -o kube-bench .
 
 ## Running on OpenShift 
 
-kube-bench includes a set of test files for Red Hat's OpenShift hardening guide for OCP 3.10 and 3.11. To run this you will need to specify `--version ocp-3.10` when you run the `kube-bench` command (either directly or through YAML). This config version is valid for OCP 3.10 and 3.11. 
+| OpenShift CIS Benchmark | kube-bench config |
+|---|---|---|
+| ocp-3.10| rh-0.7 |
+| ocp-3.11| rh-0.7 |
+
+kube-bench includes a set of test files for Red Hat's OpenShift hardening guide for OCP 3.10 and 3.11. To run this you will need to specify `--version ocp-3.10`. 
+
+When you run the `kube-bench` command (either directly or through YAML). This config version is valid for OCP 3.10 and 3.11. 
+
+
+`kube-bench` will map the `--version` to the corresponding CIS Benchmark version as indicate by the mapping table above.
+
+For example, if you specific:
+
+```
+kube-bench node --version ocp-3.10
+```
+`kube-bench` will map the `ocp-3.10` to CIS Bechmark version `rh-0.7`
+
+Also, you can specify `--benchmark` to run a specific CIS Benchmark version:
+
+```
+kube-bench node --benchmark rh-0.7
+```
+
+
+`controls` are also organized by Red Hat's OpenShift hardening guide version under the `cfg` directory for
+example `cfg/rh-0.7`.
+
+
+
+**Note:**  **`It is an error to specify both --version and --benchmark flags together`**
+
```

Comment:
I don't really think we need to repeat this again, it is covered enough above, so i think we could just delete it. But also note that this is talking about Red Hat's OpenShift hardening guide which is not a CIS Benchmark, so when we're talking about OpenShift it's wrong to say CIS Benchmark.